addons/survey/views/survey_survey_views.xml

```diff
@@ -18,10 +18,11 @@
                         attrs="{'invisible': [('session_state', '=', False)]}" />
                     <button name="action_end_session" string="Close Live Session" type="object"
                         attrs="{'invisible': [('session_state', 'not in', ['ready', 'in_progress'])]}" />
-                    <button name="action_test_survey" string="Test" type="object" attrs="{'invisible': [('active', '=', False)]}"/>
-                    <button name="action_print_survey" string="Print" type="object"/>
+                    <field name="question_ids" invisible="1"/>
```

Comment:
Would maybe move it with other question fields (or maybe before header), even if invisible let us order them a bit nicely :) Header is for button, not something else.

Reply:
I was told it is good practice to put the invisible field right before it is used... Should I still move it up?